src/registry_schemas/example_data/schema_data.py

```diff
@@ -2343,19 +2343,37 @@
 
 CONTINUATION_IN = {
     'business': {
-        'identifier': 'BC1234567',  # Identifier of the registered extra provincial business
+        'identifier': 'A1234567',  # Identifier of the registered extra provincial business
         'taxId': '123456789'  # Existing BN Number if any
     },
     'foreignJurisdiction': {
-        'name': 'Canada',
+        'country': 'CA',
+        'region': 'AB',
         'legalName': 'HAULER SERVICES',
```

Comment:
Is this used for both the legal name of an xpro or the legal name in the foreign jurisdiction (MRAS/manual entry)?

What if the business is xpro and the name in BC is different from the name in the home jurisdiction?

Reply:
xpro legalName (name in BC) inside business object and home jurisdiction legalName in foreignJurisdiction